doc/faq.md

```diff
@@ -134,11 +138,11 @@ It may need to write scripts to parse the outputs or integrated with your local
 
 <a id="support" />
 
-### Where can I found the support ? if doc not helps.
+### Where can I find the support if doc not helps?
 
-If any question, pls make sure you have read the `doc/` first, and `LVS`'s [documents](https://github.com/alibaba/LVS/tree/master/docs) are also helpful. It's the better to have some experience about networking configuration (e.g., routing, neigbour, ...) since DPVS is kernel-bypass, basic routing, IP address configurations are needed to be setup from scratch.
+If any question, pls make sure you have read the [docs](../doc) first, and `LVS`'s [documents](https://github.com/alibaba/LVS/tree/master/docs) are also helpful. It's better to have some experiences about networking configuration (e.g., routing, neigbour, ...) since DPVS is kernel-bypass, basic routing, IP address configurations are needed to be setup from scratch.
```

Comment:
If **you have** any question, **please** 
It's better to have some experiences  experience表示经验不可数
since DPVS is kernel-bypass. **Basic** routing, IP address configurations are needed **to** setup from scratch.